pyppeteer/connection.py

```diff
@@ -21,6 +27,28 @@
 logger_session = logging.getLogger(__name__ + '.CDPSession')
```

Comment:
I wonder what the reasoning behind this was? Doesn't really matter at this point

Reply:
AFAIK It's for debug logging send/received. It could probably be nicer though.

Reply:
Yeah I'm aware, I was actually just wondering why '.CDPSession' was appended to the name of the module (none of the other modules feature this logging style)

Reply:
I left only the connection logger as CDPSession logger is redundant.